wasm.go

```diff
@@ -157,59 +157,16 @@ func decodeModule(module *ModuleConfig) (m *internalwasm.Module, err error) {
 	return
 }
 
-// HostModuleConfig are WebAssembly 1.0 (20191205) exports from the host bound to a module name used by InstantiateHostModule.
-type HostModuleConfig struct {
```

Comment:
ps I had intentionally kept "config" types out of the /wasm and /wasi packages because they are mutable and probably not very safe to share to internal code.